exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/LogAdapter.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.internal;
+
+import com.google.protobuf.UnsafeByteOperations;
+import io.opentelemetry.api.internal.OtelEncodingUtils;
+import io.opentelemetry.proto.common.v1.AnyValue;
+import io.opentelemetry.proto.logs.v1.InstrumentationLibraryLogs;
+import io.opentelemetry.proto.logs.v1.ResourceLogs;
+import io.opentelemetry.proto.logs.v1.SeverityNumber;
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.logging.data.LogRecord;
+import io.opentelemetry.sdk.resources.Resource;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/** Converter from SDK {@link LogRecord} to OTLP {@link ResourceLogs}. */
+public class LogAdapter {
```

Comment:
does this need to be `public`?

Reply:
Eventually it will since the various exporters consume these adapters from different packages.

Reply:
Please add our standard javadoc boilerplate for public internal classes:
```
 * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
 * at any time.
```

Reply:
```suggestion
public final class LogAdapter {
```